pkg/monitor/cluster/clusterversions.go

```diff
@@ -34,13 +34,20 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 		}
 	}
 
+	availableRP := ""
+	switch {
+	case version.GitCommit != mon.oc.Properties.ProvisionedBy:
+		availableRP = version.GitCommit
+	}
+
```

Comment:
I get the intention this time. naming is big odd.
`availableVersion` is clear as it tells "this" version is available for upgrade.
`availableRP`. Overall we need better name for "RP maintenance/PutOrPatch" activity.

I call our naming gurus @nilsanderselde @jim-minter how we should call actions we do in putOrpatch?